src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.csproj

```diff
@@ -34,4 +34,6 @@
     <ProjectReference Include="..\AzureDevOpsClient\AzureDevOpsClient.csproj" />
   </ItemGroup>
 
+  <ProjectExtensions><VisualStudio><UserProperties /></VisualStudio></ProjectExtensions>
```

Comment:
What's this weird line?

Reply:
probably VS going haywire

Reply:
Yea, I have no idea where this came from. VS decided to go nuts I guess.